pnpm-lock.yaml

```diff
@@ -14671,6 +14671,24 @@ packages:
       ts-node: 10.9.1(@swc/core@1.2.112)(@types/node@16.11.10)(typescript@4.9.4)
       yaml: 1.10.2
 
+  /postcss-load-config@4.0.1(postcss@8.4.21)(ts-node@10.9.1):
```

Comment:
We have v3 and v4 because we use an older version of `tailwind` on the site, which depends on v3.

```
site@0.0.1

devDependencies:
tailwindcss 2.2.19
└── postcss-load-config 3.1.0
```